sunpy/database/tables.py

```diff
@@ -374,42 +376,53 @@ def _from_fido_search_result_block(cls, sr_block, default_waveunit=None):
         """
         # All attributes of DatabaseEntry that are not in QueryResponseBlock
         # are set as None for now.
-        source = sr_block.get('Source', None)
-        provider = sr_block.get('Provider', None)
-        physobs = sr_block.get('Physobs', None)
+        source = sr_block.get('Source')
+        provider = sr_block.get('Provider')
+        physobs = sr_block.get('Physobs')
         if physobs is not None:
             physobs = str(physobs)
-        instrument = sr_block.get('Instrument', None)
-        time_start = sr_block['Time'].start.datetime
-        time_end = sr_block['Time'].end.datetime
+        instrument = sr_block.get('Instrument')
+        time_start = sr_block.get('Start Time')
+        if time_start is not None:
+            time_start = time_start.datetime
+        time_end = sr_block.get('End Time')
+        if time_end is not None:
+            time_end = time_end.datetime
 
         wavelengths = sr_block.get('Wavelength', np.nan)
-        wavelength_temp = {}
-        if isinstance(wavelength_temp, tuple):
-            # Tuple of values
-            wavelength_temp['wavemin'] = wavelengths[0]
-            wavelength_temp['wavemax'] = wavelengths[1]
+        if isinstance(wavelengths, u.Quantity):
+            if wavelengths.isscalar:
+                wavemin = wavemax = wavelengths.to_value(u.nm, equivalencies.spectral())
```

Comment:
Is using equivalences here a behaviour change (even if it's a bug fix)?

Reply:
No, this is a new code path, the input here wasn't a Quantity before. In the removed block below we were always using the spectral equivalency to convert to nm.